services/minespace-web/webpack.parts.js

```diff
@@ -163,13 +163,13 @@ exports.loadCSS = ({ include, exclude } = {}) => ({
                   "border-color-base": "#003366",
                   "border-color-split": "#efefef",
                   "border-width-base": "2px",
-                  "background-color-light": "#f7f8fa",
+                  "background-color-light": "#f2f2f2",
```

Comment:
Turns out that this is where we go for changing global style variables, I feel kind of silly for not knowing/remembering before. I see you, `border-radius: 5px`. 👀